src/go/rdctl/pkg/config/config.go

```diff
@@ -51,17 +55,24 @@ var (
 	configDir         string
 	configPath        string
 	defaultConfigPath string
+	configFile        = "rd-engine.json"
+	rdSubDir          = "rancher-desktop"
 )
 
 // DefineGlobalFlags sets up the global flags, available for all sub-commands
 func DefineGlobalFlags(rootCmd *cobra.Command) {
 	var err error
+	if runtime.GOOS == "linux" && isWSLDistro() {
+		var path string
+		err , path = wslifyConfigPath()
+		fmt.Println(path)
 
+	}
```

Comment:
All you are doing is printing the AppData directory, but you are not using it, so I don't understand how this is supposed to work.

Reply:
Pushed mistakenly, it should be fixed now.